src/main/java/org/mockito/Mockito.java

```diff
@@ -452,6 +452,16 @@
  *   .thenReturn("one", "two", "three");
  * </code></pre>
  *
+ * <b>Warning</b>, if instead of chaining .thenReturn() calls, multiple when/thenReturn commands are used,
+ * then each stubbing will override the previous one:
```

Comment:
I'd rephrase to : 

```
<strong>Warning</strong> : if instead of chaining {@code .thenReturn()} calls, multiple stubbing with same matchers or arguments are used then each stubbing will override the previous one :
```